Tests/MessageTests.swift

```diff
@@ -223,7 +223,53 @@ class MessagesTests: XCTestCase {
 
         waitForExpectations(timeout: 15)
     }
-
+    
+    func testSendSimpleMessage() {
+        let ex = expectation(description: "send simple message (multipart message)")
+        
+        bobChatManager.connect(delegate: TestingChatManagerDelegate()) { bob, err in
+            XCTAssertNil(err)
+            
+            bob!.sendSimpleMessage(roomID: self.roomID, text: "simple message", completionHandler: { _, err in
+                XCTAssertNil(err)
+                
+                bob!.fetchMessagesFromRoom(bob!.rooms.first(where: { $0.id == self.roomID })!, completionHandler: { messages, err in
+                    XCTAssertNil(err)
+                    XCTAssertEqual(messages!.last?.text, "simple message")
+                    ex.fulfill()
+                })
+            })
+        }
+        
+        waitForExpectations(timeout: 15)
+    }
+    
+    func testSendMultipartMessageWithSingleTextPart() {
+        let ex = expectation(description: "send multipart message with single text part")
+        
+        bobChatManager.connect(delegate: TestingChatManagerDelegate()) { bob, err in
+            XCTAssertNil(err)
+            
+            bob!.sendMultipartMessage(
+                roomID: self.roomID,
+                parts: [
+                    PCPart(payload: PCMultipartPayload.inlinePayload(payload: PCMultipartInlinePayload(type: "text/plain", content: "hola!")))
+                ],
+                completionHandler: { messageID, err in
+                    XCTAssertNil(err)
+                
+                    bob!.fetchMessagesFromRoom(bob!.rooms.first(where: { $0.id == self.roomID })!, completionHandler: { messages, err in
+                        XCTAssertNil(err)
+                        XCTAssertEqual(messages!.last?.text, "hola!")
+                        ex.fulfill()
+                    })
+                }
+            )
+        }
+        
+        waitForExpectations(timeout: 15)
+    }
+    
```

Comment:
Can we have a test with a URL part as well? (it should end up as an attachment when you fetch it, so we have everything we need to make sure it's working.)

Reply:
Sure! Good spot.